### ISSUE TITLE
Accessibility issue: Why Terraform? audio transcript needed

### ISSUE THREAD

**Issue by jencohoon** [hashicorp/terraform#34220]
Hello Terraform Documentation Team, 

Please provide transcripts for the videos linked in the documentation.  This is a accessibility issue. 

The video "Why Terraform?" linked from this page in the TF documentation: https://developer.hashicorp.com/terraform/intro
to youtube here: https://www.youtube.com/watch?v=h970ZBgKINg
does not have a transcript.

YouTube does provide captions, but Transcripts target different user groups.
More information about the difference at the Bureau for Internet Accessibility:
https://www.boia.org/blog/why-do-i-need-both-transcripts-and-captions-for-accessibility

### Affected Pages

https://developer.hashicorp.com/terraform/intro
https://www.youtube.com/watch?v=h970ZBgKINg

### What is the docs issue?

Transcripts needed for youtube video

### Proposal

New page with link to the video and a transcript of video

**Comment by apparentlymart** [comment]
Thanks for reporting the missing transcript on this video @jencohoon, and sorry that it's missing.

As an additional note to a docs team member reading this issue, I also noticed that the captions on this video seem to be auto-generated rather than manually edited and so contain a number of errors and also lack any punctuation, which makes the dialog hard to follow without access to the audio. Hopefully we can correct both omissions at the same time, since I assume the transcript and captions will have some text common to both of them.

**Comment by crw** [comment]
Hi @jencohoon, thanks for raising this issue! I've raised it with the appropriate team. As this effort is outside of the scope of the core Terraform team, I am going to close this issue. Thanks again for this report!

**Comment by jencohoon** [comment]
Hi @crw the "Transcript" button is not more helpful than the youtube captioning system - the "Transcript" button for YouTube toggles a transcript box beside the video... 

This is more the level of transcript I was looking for - see the Transcipt below the video at: https://www.hashicorp.com/resources/what-is-infrastructure-as-code

FWIW - the "What is Terraform?" page: https://developer.hashicorp.com/terraform/intro that contains the video is part of the TF Study Guide https://developer.hashicorp.com/terraform/tutorials/certification-003/associate-study-003#review-terraform-fundamentals
On TF Study Guide the "What is Terraform?" page is linked as [Introduction to IaC](https://developer.hashicorp.com/terraform/intro) documentation

So the Education/Cert team is sending people to this video as part of the Study Guide 

Maybe pass the task to create a transcript to the Education/Cert team .

**Comment by crw** [comment]
Thanks for the added details. 

> Maybe pass the task to create a transcript to the Education/Cert team .

Indeed, this is the team to which the task has been passed and for whom I was relaying the question above.
